src/calamari/calamari.service.ts

```diff
@@ -0,0 +1,58 @@
+import { BadRequestException, HttpService, Injectable } from '@nestjs/common';
+import * as moment from 'moment';
```

Comment:
I am not sure, but this can be replaced as sugested, because there is config in tsconfig that allows that.
```suggestion
import moment from 'moment';
```